aws/resource_aws_s3_bucket.go

```diff
@@ -820,12 +820,12 @@ func resourceAwsS3BucketRead(d *schema.ResourceData, meta interface{}) error {
 	s3conn := meta.(*AWSClient).s3conn
 	ignoreTagsConfig := meta.(*AWSClient).IgnoreTagsConfig
 
-	input := &s3.HeadBucketInput{
+	input := &s3.GetBucketEncryptionInput{
```

Comment:
super small ask (this all looks great). could you add some comment lines above changes with the ticket number so that we can track these changes when we merge in more changes from upstream terraform

Reply:
Absolutely! I'll push that up in a bit.

Reply:
seems like this is the type of patch that could get lost in translation and is very important to our use case

Reply:
Fixed in e7e73cd